src/box/relay.h

```diff
@@ -100,6 +100,10 @@ relay_last_row_time(const struct relay *relay);
 double
 relay_txn_lag(const struct relay *relay);
 
+/** Returns the last sync received in response from a replica. */
+uint64_t
+relay_known_vclock_sync(const struct relay *relay);
```

Comment:
Unused?

Reply:
Yep, but I thought it would be fine to have it for later.
Removed.